src/main/java/com/southernartsoap/model/UserModel.java

```diff
@@ -0,0 +1,34 @@
+package com.southernartsoap.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
```

Comment:
add imports for

import javax.validation.constraints.Email;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.Pattern;